sirepo/template/controls.py

```diff
@@ -144,35 +144,43 @@ def _generate_madx(v, data):
     def _format_header(el_id, field):
         return f'el_{el_id}.{field}'
 
-    def _set_opt(el, field, kicker):
-        count = len(kicker.kick)
-        kicker.kick.append(el[field])
+    def _set_opt(el, field, all_orrectors):
```

Comment:
typo `all_correctors`. I make typos like this all of the time. If you ever find a spellchecker that works well with code please share it!